src/common/AndroidLauncher.ts

```diff
@@ -41,9 +41,10 @@ export class AndroidLauncher {
         targetApp: string,
         appConfig: AndroidAppPreviewConfig | undefined,
         serverPort: string,
-        targetingLwrServer: boolean = false
+        targetingLwrServer: boolean = false,
+        logger?: Logger
     ): Promise<void> {
-        const preferredPack = await AndroidUtils.fetchSupportedEmulatorImagePackage();
+        const preferredPack = await AndroidUtils.fetchSupportedEmulatorImagePackage(undefined, logger);
```

Comment:
Some of our methods currently take an optional `apiLevel`.... but we don't need `apiLevel` and have a separate work item for removing `apiLevel` and any associated logic from our code so later this will get clean up more.